packages/kernel-6.12/kernel-6.12.spec

```diff
@@ -387,12 +407,21 @@ mkdir -p %{buildroot}%{_cross_datadir}/xfsprogs/mkfs
 ln -s lts_6.12.conf %{buildroot}%{_cross_datadir}/xfsprogs/mkfs/default.conf
 
 %if "%{_cross_arch}" == "x86_64"
-# Add Neuron-related drop-ins to load the module when the hardware is present.
-mkdir -p %{buildroot}%{_cross_unitdir}/sysinit.target.d
-install -p -m 0644 %{S:220} %{buildroot}%{_cross_unitdir}/sysinit.target.d/neuron.conf
+# Add Neuron-related configuration files to load the module when the hardware is present.
+install -d 0644 %{buildroot}%{_cross_tmpfilesdir}
+sed \
+  -e "s|__KERNEL_VERSION__|%{kmajor}|" \
+  -e "s|__PREFIX__|%{_cross_prefix}|" %{S:220} > neuron.conf
+install -p -m 0644 neuron.conf %{buildroot}%{_cross_tmpfilesdir}/
+install -d 0644 %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/drivers
+sed -e 's|__NEURON_MODULES__|%{_cross_datadir}/neuron|' %{S:221} > \
+  neuron-inf1.toml
+install -m 0644 neuron-inf1.toml %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/drivers
```

Comment:
nit: could you please add a comment to separate when the next driver configurations begin?